bucket/bitwarden.json

```diff
@@ -14,6 +14,7 @@
         }
     },
     "bin": "Bitwarden.exe",
+    "post_install": "Remove-Item \"$dir\\resources\\app-update.yml\" -Recurse",
```

Comment:
Recurse flag is not required for one file?

Reply:
Right